docs/src/cli/delegate-stake.md

```diff
@@ -0,0 +1,136 @@
+# Earn Staking Rewards
+
+After you have [recieved SOL](transfer-tokens.md), you might consider putting
+it to use by delegating stake to a validator node. Solana is a Proof of Stake
+blockchain, which means that holding tokens gives you the option to influence
+which validator nodes participate in securing the ledger. The more tokens you
+hold, the more influence you have. The running assumption of any Proof of Stake
+blockchain is that the more tokens you hold, the more you have to lose if a
+validator node acts maliciously.
+
+As a tokenholder, it is your option to use your tokens as *stake*. Stake is
+what we call tokens when they reside in a *stake account*. From the stake
```

Comment:
"As a tokenholder, it is your option to use your tokens as stake."
What is the sentence meant to communicate that isn't already said in the first paragraph?

"Stake is what we call tokens when they reside in a *stake account*."
Not crazy about this sentence. How about "Tokens start acting as stake when they are moved into a *stake account*"

Reply:
Rewrote that paragraph